src/utils/schemas.js

```diff
@@ -160,6 +160,17 @@ const commentSchema = Joi.object()
     }
   });
 
+const searchRequestsSchema = Joi.object().keys({
+  travelDate: Joi.date().required(),
+  returnDate: Joi.date().required(),
```

Comment:
Great work @RaymondMwaura, however,  I think this information doesn't have to be required.

Reply:
I have updated the code, now dates are optional. Only the searchString is required.